zephyr/CMakeLists.txt

```diff
@@ -128,6 +128,12 @@ if(CONFIG_LIBOQS)
                 set(OQS_ENABLE_SIG_MAYO OFF)
         endif()
 
+        if(CONFIG_LIBOQS_ENABLE_SIG_CROSS)
```

Comment:
Zephyr seems quite sensitive to stack usage, which may be a reason for the failing build. See for example the MAYO configuration which disables MAYO_5 for this reason.

Reply:
Thank you @bhess! I disabled in Zephyr the 6 variants of CROSS that were already marked as large-stack-usage in the docs, Zephyr tests are passing now.